01-js-intro-and-jquery/08/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="x-ua-compatible" content="ie=edge" />
+	<title>Practice - 08</title>
+	<meta name="viewport" content="width=device-width,initial-scale=1" />
+	<meta name="author" content="Braulio Evangelisti" />
+	<meta name="description" content="Write response in section." />
+	<link rel="stylesheet" href="css/styles.css?v=1.0" />
+</head>
+<body>
+	
+	<header>
+		<h1>Page title</h1>
+	</header>
+	
+	<main>
+		<section class="hidden">
+			<h2>Main container</h2>
+			<p id="response">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Similique deleniti corrupti ea animi amet explicabo, provident tenetur perferendis, aliquid rem minima iste? Temporibus blanditiis aut amet, minima corporis. Commodi, id.</p>
```

Comment:
IDs usages are applied correctly but not necessary though... Just to keep it as a good practice, try to always use classes unless you really need IDs.